### PR TITLE
[Edit] Python Strings `.count()` Method

### PR DESCRIPTION
**Pull request by KTom101**
### Description

Edit to [Strings: `.count()` method](https://www.codecademy.com/resources/docs/python/strings/count). It is currently written for Arrays which already exists.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/count/count.md [hunk 1]
```diff
@@ -13,31 +13,41 @@ CatalogContent:
   - 'paths/analyze-data-with-python'
 ---
 
-Takes in a list of values of any data type, and returns the number of times(count) a particular value is present within the list.
+The **.count()** finds the number of times the specified substring occurs within a given string.
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/count/count.md:16]:
```suggestion
The **`.count()`** method finds the number of times the specified substring occurs within a given string.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@KTom101 I did a first review and only had one comment.

**Comment by KTom101** [comment]
@SSwiniarski Made the change, thanks!

File: content/python/concepts/strings/terms/count/count.md [hunk 2]
```diff
@@ -13,31 +13,41 @@ CatalogContent:
   - 'paths/analyze-data-with-python'
 ---
 
-Takes in a list of values of any data type, and returns the number of times(count) a particular value is present within the list.
+The **`.count()`** method finds the number of times the specified substring occurs within a given string.
 
 ## Syntax
 
-```py
```
Comment (HishamT) [inline content/python/concepts/strings/terms/count/count.md:20]: This block of Python is never closed!

File: content/python/concepts/strings/terms/count/count.md [hunk 3]
```diff
@@ -13,31 +13,41 @@ CatalogContent:
...
-list.count()
+The `.count()` method is called on a string using the following syntax:
+
+```pseudo
+string.count(substring, start=..., end=...)
 ```
 
-## Example 1
+- `substring` is the substring to search for
```
Comment (HishamT) [inline content/python/concepts/strings/terms/count/count.md:26]:
```suggestion
- `substring` is the substring to search for.
```

File: content/python/concepts/strings/terms/count/count.md [hunk 4]
```diff
@@ -13,31 +13,41 @@ CatalogContent:
...
+- `start` and `end` are optional and respectively specifies a starting and end position for the search
```
Comment (HishamT) [inline content/python/concepts/strings/terms/count/count.md:27]:
```suggestion
`start` and `end` are optional and specify the search's starting and end positions.
```

File: content/python/concepts/strings/terms/count/count.md [hunk 5]
```diff
@@ -13,31 +13,41 @@ CatalogContent:
...
 
-Use `.count()` to count the number of occurrences of the string `"cat"` in the `pets` list:
+> **Note:** When not specified, `start` defaults to `0` and `end` defaults to `len(string)` which means the search will be performed on the whole string.
 
-```codebyte/python
```
Comment (HishamT) [inline content/python/concepts/strings/terms/count/count.md:28]: I can't find the closing ``` for this code block.

File: content/python/concepts/strings/terms/count/count.md [hunk 6]
```diff
@@ -13,31 +13,41 @@ CatalogContent:
   - 'paths/analyze-data-with-python'
 ---
 
-Takes in a list of values of any data type, and returns the number of times(count) a particular value is present within the list.
+The **`.count()`** method finds the number of times the specified substring occurs within a given string.
```
Comment (HishamT) [inline content/python/concepts/strings/terms/count/count.md:16]: Should there be a mention of case sensitivity?

**Review by HishamT — CHANGES_REQUESTED** [review]
I have reviewed this entry and left some comments.

File: content/python/concepts/strings/terms/count/count.md (hunk 2 above)
Comment (KTom101) [reply content/python/concepts/strings/terms/count/count.md:20]: @HishamT Are you looking at this on the mobile app? It shows everything as combined but all this was removed.

**Comment by KTom101** [comment]
@HishamT I added a note for two of the changes. The others I applied.

File: content/python/concepts/strings/terms/count/count.md (hunk 5 above)
Comment (KTom101) [reply content/python/concepts/strings/terms/count/count.md:28]: @HishamT Look at my previous comment.

File: content/python/concepts/strings/terms/count/count.md (hunk 2 above)
Comment (HishamT) [reply content/python/concepts/strings/terms/count/count.md:20]: No, I am on my computer. I can't see the closing, but I will take your word for it. :grin:

**Review by HishamT — APPROVED** [review]
LGTM!

**Pull request merged**